client.go

```diff
@@ -51,13 +51,22 @@ type InternalClient interface {
 	Client
 
 	ClaimActualLRP(logger lager.Logger, traceID string, key *models.ActualLRPKey, instanceKey *models.ActualLRPInstanceKey) error
-	StartActualLRP(logger lager.Logger, traceID string, key *models.ActualLRPKey, instanceKey *models.ActualLRPInstanceKey, netInfo *models.ActualLRPNetInfo, internalRoutes []*models.ActualLRPInternalRoute, metricTags map[string]string, routable bool) error
+	StartActualLRP(logger lager.Logger,
+		traceID string,
+		key *models.ActualLRPKey,
+		instanceKey *models.ActualLRPInstanceKey,
+		netInfo *models.ActualLRPNetInfo,
+		internalRoutes []*models.ActualLRPInternalRoute,
+		metricTags map[string]string,
+		routable bool,
+		availabilityZone string,
+	) error
 	CrashActualLRP(logger lager.Logger, traceID string, key *models.ActualLRPKey, instanceKey *models.ActualLRPInstanceKey, errorMessage string) error
 	FailActualLRP(logger lager.Logger, traceID string, key *models.ActualLRPKey, errorMessage string) error
 	RemoveActualLRP(logger lager.Logger, traceID string, key *models.ActualLRPKey, instanceKey *models.ActualLRPInstanceKey) error
 
 	EvacuateClaimedActualLRP(lager.Logger, string, *models.ActualLRPKey, *models.ActualLRPInstanceKey) (bool, error)
-	EvacuateRunningActualLRP(lager.Logger, string, *models.ActualLRPKey, *models.ActualLRPInstanceKey, *models.ActualLRPNetInfo, []*models.ActualLRPInternalRoute, map[string]string, bool) (bool, error)
+	EvacuateRunningActualLRP(lager.Logger, string, *models.ActualLRPKey, *models.ActualLRPInstanceKey, *models.ActualLRPNetInfo, []*models.ActualLRPInternalRoute, map[string]string, bool, string) (bool, error)
```

Comment:
Do we have any idea why the parameters for the `{Evacuate,Remove}...ActualLRP` functions have unnamed variables while the other functions don't? 
Even in the [actual implementation](https://github.com/cloudfoundry/bbs/blob/8a30f5a0e17b69f3e149fb0c51dfaee39a29c567/client.go#L533) we name the variable for the traceid, `traceID string`.

Should this be instead:
```diff
- EvacuateRunningActualLRP(lager.Logger, string, *models.ActualLRPKey, *models.ActualLRPInstanceKey, *models.ActualLRPNetInfo, []*models.ActualLRPInternalRoute, map[string]string, bool, string) (bool, error)
+ EvacuateRunningActualLRP(logger lager.Logger, traceID string, key *models.ActualLRPKey, instanceKey *models.ActualLRPInstanceKey, netInfo *models.ActualLRPNetInfo, internalRoutes []*models.ActualLRPInternalRoute, metricTags map[string]string, routable bool, availabilityZone string) (bool, error)
```

Reply:
I don't think there is any reason those functions have unnamed variables. Both formats are fine for interface, but it is good to be consistent. I didn't want to make more changes than needed but can update the rest of the functions for consistency.